CrowdFundingApplication/CrowdFundingApplication.csproj

```diff
@@ -15,4 +15,8 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="wwwroot\Avatars\" />
```

Comment:
This folder is not accessible via a link without an authorization?

Reply:
Yes, not accessible
This folder just store user avatars

Reply:
Has been edited